.github/workflows/kind_e2e.yaml

```diff
@@ -133,14 +133,14 @@ jobs:
       if: github.base_ref == 'master'
       run: |
         helm repo add dapr https://dapr.github.io/helm-charts/ && helm repo update && export RUNTIME_VERSION=$(helm search repo dapr/dapr --devel --versions | awk '/dapr\/dapr/ {print $3; exit}' )
```

Comment:
Unrelated, but shall we also change this to use cURL like self hosted e2e?

Reply:
IMO we can leave this as it since this looks cleaner than cURL magic, but I am fine to change as well. @mukundansundar

Reply:
if we can make it consistent lets do it. But in a separate PR please.

Reply:
This is done via #1140